frontend/src/bundles/auth/store/slice.ts

```diff
@@ -2,17 +2,19 @@ import { createSlice } from '@reduxjs/toolkit';
 
 import { DataStatus } from '~/bundles/common/enums/enums.js';
 import { type ValueOf } from '~/bundles/common/types/types.js';
-import { type UserSignInResponseDto } from '~/bundles/users/users.js';
+import { type UserGetCurrentResponseDto } from '~/bundles/users/users.js';
 
-import { signIn, signUp } from './actions.js';
+import { loadCurrentUser, signIn, signUp } from './actions.js';
 
 type State = {
-    dataStatus: ValueOf<typeof DataStatus>;
-    user: UserSignInResponseDto | null;
+    authDataStatus: ValueOf<typeof DataStatus>;
+    userDataStatus: ValueOf<typeof DataStatus>;
```

Comment:
Why do you need two data statuses?

Reply:
I needed that when retrieving getUser inside sign-in, I'll refactor it as well.